internal/resourcemapping/resourcemapping.go

```diff
@@ -122,6 +123,10 @@ var (
 			},
 			awsAccount: {otelKeys: []string{string(semconv.CloudAccountIDKey)}},
 		},
+		bmsInstance: {
+			location:   {otelKeys: []string{string(semconv.CloudRegionKey)}},
```

Comment:
Could location use the zone as well? Many others resources check for zone then region